speech/cloud-client/transcribe_multichannel.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Speech API sample that demonstrates enhanced models
+and recognition metadata.
+
+Example usage:
+    python transcribe_multichannel.py resources/audio.raw
+    python transcribe_multichannel.py \
+        gs://cloud-samples-tests/speech/Google_Gnome.wav
```

Comment:
Should we put this in the storage-samples project?

Reply:
done